go.mod

```diff
@@ -30,7 +30,7 @@ require (
 	github.com/line/iavl/v2 v2.0.0-init.1.0.20210602045707-fddfe1f85001
 	github.com/line/ostracon v0.34.9-0.20210930060702-30b70e254d83
 	github.com/line/tm-db/v2 v2.0.0-init.1.0.20210824011847-fcfa67dd3c70
-	github.com/line/wasmvm v0.14.0-0.8.0
+	github.com/line/wasmvm v0.14.0-0.8.0.0.20220225061515-205ced7dc680
```

Comment:
It will be changed to squash merged commit of PR https://github.com/line/wasmvm/pull/55